bundles/plugins/org.bonitasoft.studio.common.diagram/src-test/java/org/bonitasoft/studio/common/diagram/dialog/OpenNameAndVersionDialogTest.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2015 Bonitasoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bonitasoft.studio.common.diagram.dialog;
+
+import static org.bonitasoft.studio.model.process.builders.MainProcessBuilder.aMainProcess;
+
+import org.bonitasoft.studio.assertions.StatusAssert;
+import org.bonitasoft.studio.common.repository.model.IRepositoryStore;
+import org.bonitasoft.studio.swt.rules.RealmWithDisplay;
+import org.eclipse.core.databinding.UpdateValueStrategy;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OpenNameAndVersionDialogTest {
+
```

Comment:
Shouldn't we have also a test with the other case mentioned by the bug "My \\ Diagram"?